nilearn/glm/first_level/first_level.py

```diff
@@ -220,6 +220,37 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
         results = {0.0: ols_result}
 
     return labels, results
+  
+def check_trial_type(events):
```

Comment:
```suggestion
def _check_trial_type(events):
```

Probably want to keep this function private for now.